docs/workflows/sync.rst

```diff
@@ -66,6 +66,15 @@ Remote GET Response::
    instead of the whole repository. Note that it is also possible to filter a bunch of tags that
    matches defined criteria by leveraging wildcards.
 
+Some registries contain signed images. Such registries provide signatures in different ways.
+If a registry provides signatures via its SigStore, a URL to it should be specified in
+the ``sigstore`` field when creating a Remote.
+
+.. note::
+    Repositories can support the docker API extension for signatures, or have `cosign` signatures.
```

Comment:
Some registries provide the docker API extension for `atomic container signature`type , or have `cosign type  signatures`.